Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/impl/NetworkTransformer.java

```diff
@@ -35,45 +28,60 @@
 public class NetworkTransformer implements INetworkTransformer {
 
     @Autowired
-    private INetworkManager networkManager;
-
-    @Autowired
-    private IConceptCollectionManager conceptCollectionManager;
+    private EventParser parser;
 
     @Autowired
-    private EventParser parser;
+    private ElementEventTypeDownloadService elementEventTypeDownloadService;
 
     private static final Logger logger = LoggerFactory
             .getLogger(NetworkTransformer.class);
 
+    private static int NUM_THREADS = 4;

```

Comment:
Let's make this configurable. Depending on the capabilities of the machine, admins might want to adjust this.